components/library/tntnet/tntnet-server.p5m

```diff
@@ -0,0 +1,37 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov
+#
+
+<transform file path=usr.*/man/.+ -> default mangler.man.stability uncommitted>
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)/server@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY) - appserver"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/bin/$(MACH64)/tntnet
+file path=usr/bin/tntnet
+
+file path=usr/share/man/man8/tntnet.8
+file path=usr/share/man/man7/tntnet.properties.7
+file path=usr/share/man/man7/tntnet.xml.7
+
+file path=usr/share/tntnet/tntnet-conf2xml.pl
+
+file path=usr/etc/init.d/tntnet
```

Comment:
Thanks... probably sysconfdir missing... this is where PR's are useful rather than annoying ;)
Currently the project offers no integration with SMF, systemd etc. by itself; I am not sure that "initial commit" should add that?

Other than that, the SMF wrapper would not be extremely difficult to make, even with multi-instance support ;)